### ISSUE TITLE
Use with specific roles?

### ISSUE THREAD

**Issue by nratter** [jaredonline/google-authenticator#43]
Is it possible to use this gem for a specific 'role' if I have a single User table with multiple roles?

**Comment by jaredonline** [comment]
Yep absolutely. Assuming you're using Rails, you could use [the sample setup](https://github.com/jaredonline/google-authenticator#sample-rails-setup) and just change the `before_filter` to only run on the roles you're looking for
